logistics_planning_invoicing/wizards/logistics_schedule_existing_account_move_wizard_views.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="view_logistics_schedule_existing_account_move_wizard" model="ir.ui.view">
+        <field name="name">logistics.schedule.existing.account.move.wizard.tree</field>
+        <field name="model">logistics.schedule.existing.account.move.wizard</field>
+        <field name="arch" type="xml">
+            <form>
+                <sheet>
+                    <group>
+                        <field name="logistics_schedule_id"/>
+                        <field name="journal_id" required="1"/>
+                        <field name="product_id" invisible="1"/>
+                        <field name="carrier_id" invisible="1"/>
+                        <field name="domain_invoice_line_ids" invisible="1"/>
+                        <field name="invoice_line_id" 
+                            context="{'logistics_planning_invoicing_existing': True}" 
+                            domain="[('id', 'in', domain_invoice_line_ids)]"
+                            required="1"/>
+                    </group>
+                    <footer>
+                        <button name="add_invoice" string="Add Existing Invoice" type="object" class="oe_highlight"/>
```

Comment:
Better this:
```suggestion
                        <button name="add_invoice" string="Add to Existing Invoice" type="object" class="oe_highlight"/>
```
And then, for `es` translation, the right text should be "Añadir **_a_** factura existente".